src/Cache/DefaultCache.php

```diff
@@ -282,7 +282,7 @@ private function buildCollectionCacheKey(
             $ownerIdentifier = $this->toIdentifierArray($metadata, $ownerIdentifier);
         }
 
-        return new CollectionCacheKey($metadata->rootEntityName, $association, $ownerIdentifier);
+        return new CollectionCacheKey($metadata->rootEntityName, $association, $ownerIdentifier, '');
```

Comment:
We're in `DefaultCache.php`, so let's use the default value.

```suggestion
        return new CollectionCacheKey($metadata->rootEntityName, $association, $ownerIdentifier);
```